client/src/components/Footer.js

```diff
@@ -5,6 +5,8 @@ import AddQuestion from '../icons/AddQuestion';
 import Button from './Button';
```

Comment:
Do you use this Button?

Reply:
It's there (invisible behind the Logo) but hasn't got its functionality yet